src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -69,22 +69,27 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
     seatedAtTable && venue?.id
   );
 
-  const isReactionsAudioDisabled = !venue.isReactionsMuted;
+  const defaultReactionsValue =
+    venue.isReactionsMuted ?? DEFAULT_REACTIONS_AUDIBLE;
+  const isReactionsAudioDisabled =
+    venue.isReactionsMuted ?? defaultReactionsValue
+      ? false
+      : DEFAULT_REACTIONS_AUDIBLE;
```

Comment:
```suggestion
  const defaultReactionsValue =
    venue.isReactionsMuted ?? DEFAULT_REACTIONS_AUDIBLE;
  const isReactionsAudioDisabled =
    defaultReactionsValue
      ? false
      : DEFAULT_REACTIONS_AUDIBLE;
```

Reply:
done facea7d
Also, flipped the `isReactionsMuted` initial value and done some changes to variables so they have more correct names + minor formatting